instrumentation/aws_sdk/README.md

```diff
@@ -36,8 +36,10 @@ end
 This instrumentation offers the following configuration options: 
 * `:inject_messaging_context` (default: `false`): When set to `true`, adds context key/value 
  to Message Attributes for SQS/SNS messages.
-* `suppress_internal_instrumentation` (default: `false`): When set to `true`, any spans with 
- span kind of `internal` are suppressed from traces.
+* `:enable_internal_instrumentation` (default: `false`): When set to `true`, any spans with 
+ span kind of `internal` are traced.
+* `:suppress_internal_instrumentation`: **Deprecated**. This configuration has been 
+ deprecated in a favor of `:enable_internal_instrumentation`
```

Comment:
It looks like you are removing this option altogether resulting in a breaking change. 

If you want to treat it as deprecated, as opposed to removed, I would recommend leaving the option present and giving precedence to the new option.

Reply:
Updated pr, keeping deprecated config and added warn if it is set true (since it indicates to me that's its actively being used).